packages/components/src/templates/toast-notification.stories.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../solid-components';
+import { html } from 'lit';
+import { userEvent } from '@storybook/test';
+
+export default {
+  title: 'Templates/Toast Notification',
```

Comment:
Why is this a template? I would see this in the docs as we miss the toasts ATM.

Reply:
Moved into the docs.